components/pagination/style/index.less

```diff
@@ -115,9 +115,9 @@
     color: @text-color;
     border-radius: @border-radius-base;
     list-style: none;
-    min-width: 28px;
-    height: 28px;
-    line-height: 28px;
+    min-width: @input-height-base;
```

Comment:
Just read the diff https://github.com/ant-design/ant-design/pull/4175/files it's still here.